lib/upload.js

```diff
@@ -74,6 +74,36 @@ class Upload {
     this._offsetBeforeRetry = 0;
   }
 
+  static terminate(url, options, cb) {
```

Comment:
Please add documentation to the README for the two new public methods.

Reply:
added